### PR TITLE
Update Zscaler ZPA readme.md

### PR DESCRIPTION
**Pull request by cherryleaf**
Markdown tidy up

Type of change
- Enhancement

## What does this PR do?

Tidy up of the markdown in the readme file. 

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

File: packages/zscaler_zpa/docs/README.md [hunk 1]
```diff
@@ -10,9 +9,11 @@ ECS fields where applicable and the remaining fields are written under
 ## Setup steps
 
 1. Enable the integration with the TCP input.
+
 2. Configure the Zscaler LSS Log Receiver to send logs to the Elastic Agent
-that is running this integration. See [_Setup Log Receiver_](https://help.zscaler.com/zpa/configuring-log-receiver). Use the IP address/hostname of the Elastic Agent as the 'Log Receiver Domain or IP Address', and use the listening port of the Elastic Agent as the 'TCP Port' on the _Add Log Receiver_ configuration screen.
-3. *Please make sure to use the given response formats.*
+    that is running this integration. See [Setup Log Receiver](https://help.zscaler.com/zpa/configuring-log-receiver). Use the IP address/hostname of the Elastic Agent as the 'Log Receiver Domain or IP Address', and use the listening port of the Elastic Agent as the 'TCP Port' on the _Add Log Receiver_ configuration screen.
+
+>  Note: Please make sure to use the given response formats.
```
Comment (efd6) [inline packages/zscaler_zpa/docs/README.md:16]:
```suggestion
Note: Please make sure to use the provided response formats.
```

Comment (GinnyGalicia) [reply packages/zscaler_zpa/docs/README.md:16]: If the > is used, it should generate a Markdown admonition note box.

Comment (cherryleaf) [reply packages/zscaler_zpa/docs/README.md]: I've changed this to Elastic's `<DocCallOut title="Note">` tag

**Review by bmorelli25 — CHANGES_REQUESTED** [review]
The manifest version needs to be updated to `0.2.1`

File: packages/zscaler_zpa/docs/README.md (hunk 1 above)
Comment (bmorelli25) [reply packages/zscaler_zpa/docs/README.md]: Unfortunately the `<DocCallOut title="Note">` don't work in `.md` files and are specifically for our `.mdx` files. Can you change this back please?

**Comment by cherryleaf** [comment]
Cannot resolve

**Pull request closed without merging**